config/routes.rb

```diff
@@ -157,6 +157,11 @@
     end
 
     resources :data_requests
+
+    resource :letters, only: [], path: "letters/:type"  do
+        get "/new", to: 'letters#new'
+        get "/render", to: 'letters#render_letter', as: 'render'
```

Comment:
Should `render_letter` be `show` ?

Reply:
Excellent wheeze, never occurred to me. I knew it wasn't create...